kalliope/stt/cmusphinx/README.md

```diff
@@ -0,0 +1,24 @@
+### CMU Sphinx
+
+The CMU Sphinx STT is based on the [CMUSPhinx engine](http://cmusphinx.sourceforge.net/wiki/).
```

Comment:
Can you also add in the doc that it is a self hosted solution? many people required that feature.